### PR TITLE
Adding ADR: proposed for more scalable architecture 

### PR DESCRIPTION
**Pull request by bradleyandrick**
**Related Issue(s):**

- NA

**Proposed Changes:**

1. Added first ADR for decision on moving to scalable frontend architecture 🥳 

**PR Checklist:**

- [X] I have added my changes to the [CHANGELOG](https://github.com/Element84/filmdrop-ui/blob/main/CHANGELOG.md) **or** a CHANGELOG entry is not required.

### CODE REVIEW

File: ADRs/adr_config_refactor_11.8.23.md [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+        }
+      ]
+    }
+  ]
+}
+```
+
+## Proposed Config.json Structure:
+
+```json
```
Comment (philvarner) [inline ADRs/adr_config_refactor_11.8.23.md:179]: changing these blocks to `jsonc` should allow them to render correctly

File: ADRs/adr_config_refactor_11.8.23.md [hunk 2]
```diff
@@ -0,0 +1,306 @@
...
+      },
+      "CUSTOM_APP_2": {
+        // all settings for custom app here
+      }
+    }
+  }
+}
+```
+
+## Exiting Directory Structure:
```
Comment (philvarner) [inline ADRs/adr_config_refactor_11.8.23.md:220]:
```suggestion
## Existing Directory Structure:
```

File: ADRs/adr_config_refactor_11.8.23.md (hunk 1 above)
Comment (bradleyandrick) [reply ADRs/adr_config_refactor_11.8.23.md]: thank you!, fixed

File: ADRs/adr_config_refactor_11.8.23.md [hunk 3]
```diff
@@ -0,0 +1,306 @@
...
+
+```json
```
[the remark this answers is not in the document]
Comment (bradleyandrick) [reply ADRs/adr_config_refactor_11.8.23.md]: removed

**Review by philvarner — APPROVED** [review]

**Pull request merged**